dangerfile.ts

```diff
@@ -73,14 +73,17 @@ const correspondingTestsForAppFiles = touchedAppOnlyFiles.map(f => {
 
 // New app files should get new test files
 // Allow warning instead of failing if you say "Skip New Tests" inside the body, make it explicit.
-const testFilesThatDontExist = correspondingTestsForAppFiles.filter(f => fs.existsSync(f))
+const testFilesThatDontExist = correspondingTestsForAppFiles
+                                 .filter(f => !f.includes("index-tests.tsx")) // skip indexes
+                                 .filter(f => !fs.existsSync(f))
```

Comment:
The check for whether the file exists was wrong it was filtering only when it did exist